composer.json

```diff
@@ -46,14 +46,15 @@
         "symfony/phpunit-bridge": "^5.3",
         "symfony/polyfill-uuid": "^1.20",
         "symfony/process": "5.3.*",
+        "symfony/runtime": "5.3.*",
```

Comment:
Why is it installed now?

Reply:
This is because of this PR : https://github.com/symfony/recipes/commit/9a90e6978504aabb3e897e65d485c7d02f792462

Symfony runtime is now a depandancy of the Symfony console.

Reply:
(An issue already exists : https://github.com/symfony/recipes/issues/992)